### ISSUE TITLE
Subtitles are not displaying (ERR: impossible to display subtitles.)

### ISSUE THREAD

**Issue by kirandcunha** [videojs/videojs-contrib-dash#365]
## Description
stpp subtitles embedded in dash manifest are not displayed. This is reproducible in [sample video player](http://videojs.github.io/videojs-contrib-dash/).

## Steps to reproduce
Play URL https://livesim.dashif.org/dash/vod/testpic_2s/multi_subs.mpd using videojs-contrib-dash

### Expected
Subtitles should display

### Actual
Subtitles are not displaying

### Error output
[257263][TextTracks] impossible to display subtitles. 

## Additional Information
Same URL plays fine in [reference dash player](http://reference.dashif.org/dash.js/latest/samples/dash-if-reference-player/index.html).

#### videojs
7.14.3

#### browsers
Tested on Chrome, Edge (Chromium) and Firefox and issue exists in all

#### OSes
Tested on Windows 10

### plugins
No

**Comment by gkatsev** [comment]
Can confirm it's an issue. Not really sure what's up.

**Comment by rudemateo** [comment]
I'm seeing the same error with our `stpp` captions. Note: the stream / captions display fine in the 4.0.1 DASH reference player

**Comment by rudemateo** [comment]
potential "fix" mentioned here: https://github.com/videojs/videojs-contrib-dash/issues/236#issuecomment-484205633

**Comment by gkatsev** [comment]
Fixed via #319.

**Comment by gkatsev** [comment]
Just published videojs-contrib-dash 5.1.0 which allows for TTML captions if set up via `useTTML` option
```js
videojs('video', {
  html5: {
    dash: {
      useTTML: true
    }
  }
});
```
